### DISCUSSION TITLE
Checksum failure with Panoply

### DISCUSSION THREAD

**Discussion by mathomp4** [Homebrew/discussions#1598] [Casks]
I seem to be having an issue upgrading my `panoply` cask:
```console
❯ brew upgrade panoply
==> Upgrading 1 outdated package:
panoply 4.12.6 -> 4.12.7
==> Upgrading panoply
==> Caveats
panoply requires Java 8+. You can install the latest version with:
  brew install --cask adoptopenjdk

==> Downloading https://www.giss.nasa.gov/tools/panoply/download/PanoplyMacOS-4.12.7.dmg
Already downloaded: /Users/mathomp4/Library/Caches/Homebrew/downloads/4c7a5c342d236447da3f0d63d23cf150b33b2be95136de83e4dc7ab684702d77--PanoplyMacOS-4.12.7.dmg
==> Purging files for version 4.12.7 of Cask panoply
Error: panoply: SHA256 mismatch
Expected: 36243fdeb58fcd75d3a90655a1ab4b070228ac2396650487b9d3a582739cb17f
  Actual: 87bc04c0b2ddf2470bd546f9e96c2163e8653e195860c45e7bb2154b4d26aaa9
    File: /Users/mathomp4/Library/Caches/Homebrew/downloads/4c7a5c342d236447da3f0d63d23cf150b33b2be95136de83e4dc7ab684702d77--PanoplyMacOS-4.12.7.dmg
To retry an incomplete download, remove the file above.
```
When I delete the file as suggested, the same error occurs. 

I also tried downloading Panoply *directly* from [its site](https://www.giss.nasa.gov/tools/panoply/download/) and that too leads me to a different hash:
```console
❯ sha256sum PanoplyMacOS-4.12.7.dmg
87bc04c0b2ddf2470bd546f9e96c2163e8653e195860c45e7bb2154b4d26aaa9  PanoplyMacOS-4.12.7.dmg
```
than is in [the Panoply formula](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/panoply.rb):
```ruby
cask "panoply" do
  version "4.12.7"
  sha256 "36243fdeb58fcd75d3a90655a1ab4b070228ac2396650487b9d3a582739cb17f"
...
```
I think this is a bug (though the [bump-cask-pr PR](https://github.com/Homebrew/homebrew-cask/pull/106665) worked), but the issue template says: 

> If you’re sure it’s reproducible and not just your machine, submit an issue so we can investigate

Well...I only have one Mac to test this on, so... 🤷🏼‍♂️ 

Is it reproducible for anyone else out there? If so, then I feel an issue can be filed.

**Comment by nandahkrishna** [comment]
I'm able to reproduce this. Please take a look at https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/checksum_does_not_match_error.md, you may want to open a PR to fix this.

Note: [This link](https://www.giss.nasa.gov/tools/panoply/download/Panoply-4.12.7.sha256.txt) shows the checksums as well, so it does seem like a checksum change is needed.

**Reply by mathomp4** [reply]
Okay. One of the [weirdest-titled PRs](https://github.com/Homebrew/homebrew-cask/pull/106703) has been filed!

**Reply by nandahkrishna** [reply]
You could just use "Update checksum for Panoply" or something similar 😅.

**Reply by mathomp4** [reply]
I got spanked once for not following exact Homebrew PR titling. So whatever bump-cask-pr says is what I'll use! 😄
